src/v2/expressions.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Why is param expression specific to v2?

Reply:
The code that currently allows users to define params in their functions code lives in the v2 namespace, so...I guess I just assumed?

Reply:
go/cf3-params-detailed is the doc that formally proposes ditching v2, but it seems easier for Victor to do that when he reaches a stopping point than having some PRs in v2 and some in root. Besides, technically that should be done in the launch.next branch, right? Technically we're changing an export path.